rosdep/python.yaml

```diff
@@ -5272,6 +5272,7 @@ python-zmq:
   gentoo: [dev-python/pyzmq]
   ubuntu:
     '*': [python-zmq]
+    focal: [python3-zmq]
```

Comment:
This is implicitly the Python 2 version of the package, so we shouldn't add focal here.  Instead, please add a new key called `python3-zmq`, and add the entry there.